public/components/trace_analytics/components/traces/traces_content.tsx

```diff
@@ -43,6 +43,8 @@ export function TracesContent(props: TracesProps) {
   const [redirect, setRedirect] = useState(true);
   const [loading, setLoading] = useState(false);
   const [trigger, setTrigger] = useState<'open' | 'closed'>('closed');
+  const isDataPrepper = mode === 'data_prepper';
```

Comment:
I was wondering if we can define this into an util file, so that we don't need to re define it in all the related classes. Maybe we can setup a util file under here: https://github.com/opensearch-project/dashboards-observability/tree/main/public/components/trace_analytics